com.microsoft.mrtk.uxcore/Experimental/NonNativeKeyboard/NonNativeKeyboard.cs

```diff
@@ -14,11 +14,10 @@ namespace Microsoft.MixedReality.Toolkit.UX.Experimental
     /// A simple general use keyboard that provides an alternative to the native keyboard offered by each device.
     /// </summary>
     /// <remarks>
-    /// <remarks>
     /// This is an experimental feature. This class is early in the cycle, it has 
     /// been labeled as experimental to indicate that it is still evolving, and 
     /// subject to change over time. Parts of the MRTK, such as this class, appear 
-    /// to have a lot of value even if the details haven�t fully been fleshed out. 
+    /// to have a lot of value even if the details haven�t fully been fleshed out. 
```

Comment:
what is the character we are supposed to for these?

Reply:
Fixed

Reply:
oops...thought I did a Find-All and Replace...  I guess I didn't...

Fixed now!